test/sandbox/contract/printEventTopics.ts

```diff
@@ -0,0 +1,23 @@
+import { web3 } from '../../../src/helpers/web3'
+import Logger from '../../../src/helpers/Logger'
+import { AbiItem } from 'web3-utils'
+
+require('dotenv').config()
+
+/**
+ *  SANDBOX: Prints events topics
+ *  RUN:     yarn sandbox test/sandbox/contract/printEventTopics.ts
+ */
+const log = new Logger('sandbox:contract/printEventTopics')
+
+async function exec (): Promise<void> {
```

Comment:
Why is this asynchronous?

Reply:
Here is not required. I'm trying to follow always the same pattern/structure for all of the "sandboxes"

This way we don't need to change anything if we make use of any async API method

Reply:
Same reason why I gave it such a generic name `exec` instead of saying what it really does

Reply:
Got it